src/views/tutorial-view/components/featured-in-collections/index.tsx

```diff
@@ -1,46 +1,46 @@
-import { getCollectionSlug } from 'views/collection-view/helpers'
-import CardLink from 'components/card-link'
-import { Collection as ClientCollection } from 'lib/learn-client/types'
-
-interface FeaturedInCollectionsProps {
-  collections: CollectionCardProps[]
-}
-
-// @TODO move this to the collection card component file when we make it
-export interface CollectionCardProps
-  extends Pick<
-    ClientCollection,
-    'id' | 'name' | 'slug' | 'theme' | 'description'
-  > {
-  numTutorials: number
-}
+import CollectionCard from 'views/tutorial-view/components/collection-card'
+import { FeaturedInCollectionsProps } from './types'
+import s from './featured-in-collections.module.css'
 
 // This should render the eventual `CollectionCard` component (doesn't exist yet)
 // which will be used on many other views
 export function FeaturedInCollections({
+  className,
   collections,
 }: FeaturedInCollectionsProps): React.ReactElement {
   if (collections.length === 0) {
     return null
   }
 
   return (
-    <>
-      <h2>Featured Collections</h2>
-      <ul>
-        {collections.map((c) => {
+    <div className={className}>
+      <h2 className={s.heading}>This tutorial also appears in:</h2>
+      <ul className={s.listRoot}>
+        {collections.map((collection) => {
+          const { id, slug, numTutorials, name, description, theme } =
+            collection
           return (
-            <li key={c.id}>
-              <CardLink href={getCollectionSlug(c.slug)}>
-                <span>{c.numTutorials} Tutorials</span>
-                <h3>{c.name}</h3>
-                <p>{c.description}</p>
-                <p>{c.theme} Logo</p>
-              </CardLink>
+            <li key={id} className={s.listItem}>
+              <CollectionCard
+                /**
+                 * Note: className here necessary because CardLink
+                 * doesn't automatically behave as one might expect.
+                 * It requires height 100% to be set on the inner <a /> element,
+                 * which we do through this className.
+                 */
```

Comment:
This comment breaks up the markup a quite a bit (makes it a little harder to read), could it be moved to live right above the `.listItemCard` class block in `src/views/tutorial-view/components/featured-in-collections/featured-in-collections.module.css`?

Reply:
✅ Moved